.flake8

```diff
@@ -0,0 +1,11 @@
+[flake8]
+max-line-length = 99
```

Comment:
The Python standard is 79, but I bumped it up since we have a lot of long strings in our tests.